src/modelApiMethods.js

```diff
@@ -0,0 +1,105 @@
+export const fetchComment = async (url, appID, character) => {
+  /* eslint-disable no-underscore-dangle */
+  const response = await fetch(`${url}${appID}/comments?item_id=${character._id}`, {
+    method: 'GET',
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  const jsonRespnse = await response.json();
+  return jsonRespnse;
+};
+
+export const commentCounter = (response) => {
+  /* eslint-disable no-else-return */
+  if (response.length > 0) {
+    return response.length;
+  } else {
+    return 0;
+  }
+};
```

Comment:
- [ ] as the previous CR already mentioned "`Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice. Instead of disabling eslint, it will be nice to fix the issue since it is a requirement for this project."
to fix this error the function might look something like below:
```
export const commentCounter = (response) => {
  if (response.length > 0) {
    return response.length;
  }
  return 0;
};
```